src/inc/gcinfodecoder.h

```diff
@@ -240,7 +240,7 @@ class BitStreamReader
         _ASSERTE( pBuffer != NULL );
 
         m_pCurrent = m_pBuffer = dac_cast<PTR_size_t>((size_t)dac_cast<TADDR>(pBuffer) & ~((size_t)sizeof(size_t)-1));
-        m_RelPos = m_InitialRelPos = (int)((size_t)dac_cast<TADDR>(pBuffer) % sizeof(size_t)) * 8;
+        m_RelPos = m_InitialRelPos = (int)((size_t)dac_cast<TADDR>(pBuffer) % sizeof(size_t)) * 8/*BITS_PER_BYTE*/;
```

Comment:
I think you can just define the BITS_PER_BYTE in this header (somewhere around the lines 40..48).

Reply:
@janvorli 
lines 40..48 are under ```#ifdef FEATURE_REDHAWK``` So I've got ```undefined symbol``` then. When I placed  it at line 17 after ```_min``` macro, I've got another story

```
[ 19%] Building CXX object src/vm/wks/CMakeFiles/cee_wks.dir/__/__/gc/gcsvr.cpp.o
In file included from /share/CORERT/coreclr/src/gc/handletablecore.cpp:19:
/share/CORERT/coreclr/src/gc/handletablepriv.h:21:9: error: 'BITS_PER_BYTE' macro redefined [-Werror,-Wmacro-redefined]
#define BITS_PER_BYTE               (8)
        ^
/share/CORERT/coreclr/src/inc/gcinfodecoder.h:17:9: note: previous definition is here
#define BITS_PER_BYTE (sizeof(uint8_t)*8)
        ^
In file included from /share/CORERT/coreclr/src/gc/handletable.cpp:21:
/share/CORERT/coreclr/src/gc/handletablepriv.h:21:9: error: 'BITS_PER_BYTE' macro redefined [-Werror,-Wmacro-redefined]
#define BITS_PER_BYTE               (8)
        ^
/share/CORERT/coreclr/src/inc/gcinfodecoder.h:17:9: note: previous definition is here
#define BITS_PER_BYTE (sizeof(uint8_t)*8)
        ^
1 error generated.
```

Reply:
Ok, let's merge it as it is then.